tubular/nominal.py

```diff
@@ -947,7 +947,10 @@ def transform(self, X):
         # Apply OHE transform
         X_transformed = OneHotEncoder.transform(self, X[self.columns])
 
-        input_columns = self.get_feature_names(input_features=self.columns)
+        if hasattr(self, "get_feature_names"):
```

Comment:
Can we add an explanatory comment here? could be confusing otherwise.

something like 'sklearn.OneHotEncoder.get_feature_names changed to sklearn.OneHotEncoder.get_feature_names_out in v1.2.  Logic to accommodate old and newer versions'